frontend/src/citizen-frontend/messages/ThreadView.tsx

```diff
@@ -142,6 +136,14 @@ export default React.memo(function ThreadView({
 
   useEffect(() => setReplyEditorVisible(false), [threadId])
 
+  const messagesEndRef = useRef<HTMLSpanElement>(null)
+  const scrollToBottom = () => {
+    messagesEndRef.current?.scrollIntoView({ behavior: 'smooth' })
+  }
+  useEffect(() => {
+    scrollToBottom()
+  })
```

Comment:
This `useEffect` should have `[]` as its dependency array so that the view isn't scrolled every time it's rerendered.

Reply:
I actually think it makes sense to scroll again on re-render, because re-render should only happen when changing threads or when a new message is sent. In both cases it makes sense to scroll again.

Reply:
Oh ok, I guess it's just preference so it's fine then 😄

Should you mark those conditions explicitly since there's no guarantee that the component doesn't rerender on some other condition in the future?

Reply:
Added `[messages, replyEditorVisible]`, which seems to work pretty well.